cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/util/AbstractSamlObjectBuilder.java

```diff
@@ -212,6 +211,9 @@ public String marshalSamlXmlObject(final XMLObject object, final StringWriter wr
         try {
             final MarshallerFactory marshallerFactory = XMLObjectProviderRegistrySupport.getMarshallerFactory();
             final Marshaller marshaller = marshallerFactory.getMarshaller(object);
+            if (marshaller == null) {
```

Comment:
Isn't it the same change than in https://github.com/Jasig/cas/pull/1032/files#diff-3eae41a3c73981deb47c12d09cffffeaR199?